contracts/ExitHandler.sol

```diff
@@ -123,6 +123,42 @@ contract ExitHandler is DepositHandler {
     );
   }
 
+  function startDepositExit(uint256 _depositId) public payable {
+    require(msg.value >= exitStake, "Not enough ether sent to pay for exit stake");
+    // check that deposit exits
+    Deposit deposit = deposits[uint32(_depositId)];
+    require(deposit.owner == msg.sender, "Only deposit owner can start exit");
+    require(deposit.amount > 0, "deposit has no value");
+    require(exits[bytes32(_depositId)].amount == 0, "The exit of deposit has already been started");
+    require(!exits[bytes32(_depositId)].finalized, "The exit for deposit has already been finalized");
+    
+    uint256 priority;
+    if (isNft(deposit.color)) {
+      priority = (nftExitCounter << 128) | uint128(_depositId);
```

Comment:
This is a bit nitpicky, but since nftExitCounter is uint256, shifting it here will delete the most significant assuming we ever get over 2^128 exits. Maybe we should just change nftExitCounter to uint128?

Reply:
> assuming we ever get over 2^128 exits
would also need to change deposit counter. I'd say nice problem to have, let's put it into a separate issue and fix it once we have > 2^10 exits/deposits.